src/foam/core/lib.js

```diff
@@ -124,7 +124,13 @@ foam.LIB = function LIB(model) {
       typeof model.constants === 'object',
       'Constants must be a map.');
 
-    for ( var key in model.constants ) root[key] = model.constants[key];
+    for ( var key in model.constants ) {
```

Comment:
I know I fixed this before somewhere but I can't find the code, no idea what happened to it.  The main limitation here is that we're one-offing a solution for constants and not methods or any other axioms that LIBs may eventually have.

A more general solution would be to call `foam.json.parse` here, or maybe to improve the non-strict mode of JSON Outputter to output literal calls to `foam.json.Outputter.create()` as opposed to `{ class: 'foam.json.Outputter' }` but there may be other issues with that.

This is okay for now but we should keep this in mind.